Formula/px4-sim-gazebo.rb

```diff
@@ -1,21 +1,8 @@
-class XRequirement < Requirement
-  fatal true
-
-  satisfy(build_env: false) { which("xquartz") }
-
-  def message
-    <<~EOS
-      XQuartz is required; install it via:
-        brew install --cask xquartz
-    EOS
-  end
-end
-
-class Px4SimGazebo < Formula
-  desc "PX4 Gazebo simulation"
+class Px4SimGazeboHarmonic < Formula
+  desc "PX4 Gazebo Harmonic simulation"
   homepage "http://px4.io"
-  url "https://raw.githubusercontent.com/PX4/PX4-Autopilot/master/Tools/px4.py"
-  version "1.11.0"
+  url "https://raw.githubusercontent.com/PX4/PX4-Autopilot/main/Tools/px4.py"
```

Comment:
The new URL is

https://raw.githubusercontent.com/PX4/PX4-Autopilot/refs/heads/main/Tools/px4.py